data/blog/rene-pickhardt-receives-lts-grant.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: "Long-Term Support For René Pickhardt"
+date: '2024-01-18'
+tags: ['OpenSats', 'grants', 'bitcoin', 'LTS']
+draft: false
+authors: ['cfunk']
+images: ['/static/images/blog/22-rene.jpg']
+summary: "OpenSats Welcomes René Pickhardt as an LTS Grantee."
+---
+
+
+We are very excited to award a Long-term Support Grant to René Pickhardt,
```

Comment:
you could link https://ln.rene-pickhardt.de (but I don't think it's important)

Reply:
We are very excited to award a Long-term Support Grant to [René Pickhardt](https://ln.rene-pickhardt.de/),